src/AssetCompiler.php

```diff
@@ -60,16 +60,14 @@ public function __construct(FilterRegistry $filters, $debug)
      */
     public function generate(AssetTarget $build)
     {
-        $filters = $this->filterRegistry->collection($build);
+        $filters = $this->filterRegistry->collection($build, $this->debug);
         $output = '';
         foreach ($build->files() as $file) {
             $content = $file->contents();
             $content = $filters->input($file->path(), $content);
             $output .= $content . "\n";
         }
-        if (!$this->debug || php_sapi_name() === 'cli') {
-            $output = $filters->output($build->path(), $output);
-        }
+        $output = $filters->output($build->path(), $output);
```

Comment:
So far output filters are minifiers, which is why they don't apply when debug is enabled.